apps/system/js/software_button_manager.js

```diff
@@ -352,9 +352,12 @@
       }
     },
 
-    // We just add this handler to prevent fuzzing issues where tapping the SHB
-    // will trigger events in the current app.
-    _emptyEventHandler: function() {
+    /**
+     * Used to prevent taps on the SHB container from stealing focus.
+     * @memberof SoftwareButtonManager.prototype
```

Comment:
let's keep a mention of fuzzing in the comment since it (hopefully) still applies.